easybuild/easyconfigs/m/meshalyzer/meshalyzer-20200308-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'MakeCp'
+
+name = 'meshalyzer'
+local_commit = 'b1ab66863a293768cd23b444f16efe00d0acef4e'
+version = '20200308'  # datestamp of commit
+versionsuffix = '-Python-3.8.2'
+
+homepage = 'https://git.opencarp.org/openCARP/meshalyzer'
+description = "Graphical program for display time dependent data on 3D finite elment meshes"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://git.opencarp.org/openCARP/meshalyzer/-/archive/%s' % local_commit]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+patches = ['meshalyzer-2.0_fix-VTK-linking.patch']
+checksums = [
+    'b1df0be12d377a4e12e0d1f2cd17820b9d5cc3a5a813a17d187ee72adf5a8500',  # meshalyzer-20200308.tar.gz
+    '23f55c2290ea9b62440bdf8e0c1006116438beb7c4b7065b8cf136387a537e0f',  # meshalyzer-2.0_fix-VTK-linking.patch
+]
+
+local_vtk_majmin_ver = '8.2'
+dependencies = [
+    ('libpng', '1.6.37'),
+    ('FLTK', '1.3.5'),
+    ('glew', '2.1.0'),
+    ('freeglut', '3.2.1'),
+    ('HDF5', '1.10.6'),
+    ('VTK', '%s.0' % local_vtk_majmin_ver, versionsuffix),
+]
+
+buildopts = 'HDF5_ROOT=$EBROOTHDF5 LIB_EXT=".so" '
```

Comment:
@boegel just checking that this shouldn't be `SHLIB_EXT`?

Reply:
@boegel ping?

Reply:
@verdurin Sorry for overlooking this, fixed now...